### PR TITLE
change playlist total time display to HH:MM:SS

### PR DESCRIPTION
**Pull request by nikmartin**
Signed code agreement as well.

### CODE REVIEW

**Comment by daschuer** [comment]
Great, thank you very much. LGTM

File: src/library/playlistfeature.cpp [hunk 1]
```diff
@@ -173,8 +173,7 @@ void PlaylistFeature::buildPlaylistList() {
             playlistTableModel.index(row, durationColumn)).toInt();
         m_playlistList.append(qMakePair(id, QString("%1 (%2) %3")
                                         .arg(name, QString::number(count),
-                                                mixxx::Duration::formatSeconds(duration))));
-    }
```
Comment (daschuer) [inline src/library/playlistfeature.cpp]: is this curly brace lost?

Comment (nikmartin) [reply src/library/playlistfeature.cpp]: I don't know how it ended up there, but it's the closing brace of line 165. :)

Comment (nikmartin) [reply src/library/playlistfeature.cpp]: Correction: You are right, when I adjusted the formatting of my addition I removed a curly brace! I had just compiled to test and before committing made one last tweak (also known as a bug). just fixed and squashed.

**Comment by daschuer** [comment]
Just tested this branch and it is working nice. LGTM Thank you.

**Pull request merged**